token-metadata/program/src/instruction/metadata.rs

```diff
@@ -896,3 +903,63 @@ impl InstructionBuilder for super::builders::Update {
         }
     }
 }
+
+/// Prints an edition from a master edition.
+///
+/// # Accounts:
+///
+///   0. `[writable]` Edition Metadata
+///   1. `[writable]` Edition
+///   2. `[writable]` Edition Mint
+///   3. `[writable]` Edition Token Account Owner
```

Comment:
Does the account owner need to be `writable`?

Reply:
It shouldn't be, good catch.